runtime/queries/ada/locals.scm

```diff
@@ -0,0 +1,33 @@
+;;  Better highlighting by referencing to the definition, for variable
+;;  references. However, this is not yet supported by neovim
```

Comment:
```suggestion
;;  references.
```

since it's unrelated. I'm not totally sure but I think neovim does support locals these days anyways